src/sidebar/components/HelpPanel.tsx

```diff
@@ -44,13 +50,15 @@ type HelpPanelTabProps = {
  */
 function HelpPanelTab({ linkText, url }: HelpPanelTabProps) {
   return (
-    <div className="flex-1 border-r last-of-type:border-r-0">
-      <Link
-        href={url}
-        classes="flex items-center justify-center space-x-2 text-color-text-light text-lg font-medium"
-        target="_blank"
-      >
-        <span>{linkText}</span> <Icon name="external" classes="w-3 h-3" />
+    <div
+      // Set this element's flex-basis per its flex parent and also establish
+      // a flex container (centered on both axes)
+      className="flex-1 flex items-center justify-center border-r last-of-type:border-r-0 text-lg font-medium"
+    >
+      <Link color="text-light" href={url} target="_blank">
+        <div className="flex items-center gap-x-2">
```

Comment:
Technically, a `Link` will provide us a `flex items-center` layout, so would could have achieved the same effect (_sans_ `div`) by setting `classes="gap-x-2"` on `Link`. However, I feel that that splits the styling context awkwardly. The extra layout `div` here ensures that future devs can see the intent.

I'll continue thinking on this and similar use cases as they do come up frequently.